pkg/evpn/svi_subnet.go

```diff
@@ -83,8 +83,8 @@ func (s *Server) CreateSubnet(_ context.Context, in *pb.CreateSubnetRequest) (*p
 			log.Printf("error: %v", err)
 			return nil, err
 		}
-		// now get VRF/VPC to plug this bridge into
-		vpc, ok := s.Vpcs[in.Subnet.Spec.VpcNameRef]
+		// now get VRF to plug this bridge into
+		vpc, ok := s.Vrfs[in.Subnet.Spec.VpcNameRef]
```

Comment:
Why a VRF is assigned to a VPC ?

Reply:
variable name is incorrect, will change, thanks for comment